src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/domain/repository/JudgeRepository.java

```diff
@@ -10,4 +10,6 @@
 @Repository
 public interface JudgeRepository extends JpaRepository<Judge, Integer> {
     List<Judge> findByTribunalOffice(TribunalOffice tribunalOffice);
+
+    boolean existsByCodeOrName(String code, String name);
```

Comment:
I think this needs to include Tribunal office. For example, if the same name is across different offices and they both had different codes, this wouldn't add in the judge because the name is the same despite it being in a different office

Reply:
Bristol - 001 - Robin
Leeds - 002 - Robin 

If you tried something like this, I think the Leeds judge wouldn't be added

Reply:
Thanks Harpreet, as discussed yesterday I have updated the logic to check "if same code AND office exists already, if not then add the judge", so name check is removed. Also updated it to return a better error message in the response entity as you have suggested.

Reply:
Name check is still required. 2 Judges in the same office cannot have the same name